src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -113,29 +113,34 @@ Here's more detail about the two authentication domain sections:
+* For the step of granting access to groups in New Relic: for some basics of how access grants work, see [Access grants](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#understand-concepts). For a tutorial on how to create access grants, see the [user management tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/#group-access).
+* Note that you won't be able to edit users or groups in New Relic: all user and group changes must be done from your identity provider. 
 
-If you’ve selected the guided workflow, we’ll match these users by email address. There will be no need to later complete the user import step for these users given they’ll have been brought into New Relic via your identity provider. When done, you can skip ahead to the **Migrate assets** step.
+When done, you can skip ahead to Step #5. 
 
 ### Login methods [#login-methods]
 
 If you chose **Manually** for [managing users](#managing-users), you have a choice for how those users log in. You can select either a) email/password login or b) single sign on (SSO). Note that SSO is available only for organizations with Pro or Enterprise edition.
 
 You can select multiple authentication methods based on your needs. For example, you might use SSO for your employees and username/password for contractors. 
 
-If you select SSO, you must complete [additional configuration steps](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#saml). You can complete the procedure for setting up SSO, and then come back to the migration wizard to continue the migration process. If you select more than one authentication method, note that you’ll need to add a new [authentication domain](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#auth-domain-definition).
+If you select SSO, you must complete [additional configuration steps](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#saml). Some tips for doing this: 
 
-Note that if you're already using an identity provider's New Relic app, you'll probably need to use a newer version made for New Relic's newer user model. For more on this, see [Identity provider apps](#idp-apps), below. 
+* If you're already using a New Relic app for either Okta, Azure, or OneLogin, you're likely using an out-of-date one. The out-of-date apps are titled "New Relic by account" while the newer apps are titled "New Relic by organization."
+* To access the new New Relic user management UI, you'll have to ensure you're logged in via your new user record: this may require logging out, logging back in, and verifying your email to see all logins associated with that email. 
```

Comment:
I'd break this compound sentence into two separate ones. GH isn't showing me that handy edit tool, so I'll just type it out here:

* To access the new New Relic user management UI, you'll have to ensure you're logged in via your new user record. This may require logging out, logging back in, and verifying your email to see all logins associated with that email.